paperqa/types.py

```diff
@@ -334,6 +334,12 @@ class DocDetails(Doc):
     doi_url: str | None = None
     doc_id: str | None = None
     file_location: str | os.PathLike | None = None
+    license: str | None = Field(
+        default=None,
+        description="string indicating license."
+        " Should refer specifically to pdf_url (since that could be preprint). None means unknown/unset.",
+    )
+    pdf_url: str | None = None
```

Comment:
What do you think of calling this `source_url` over `pdf_url`, as we'll someday generalize beyond PDFs

Reply:
I think `doi_url` should cover the HTML/WEB url and `pdf_url` should directly go to PDF. We may later add `source_url` and then I think `doi_url` would be depcretaed rather than `pdf_url`

Reply:
Sounds good